CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+## How to contribute to Decidim
+
+We're really glad you're reading this, because we need volunteer developers to help this project come to fruition.
+
+If you haven't already, come find us in [Gitter](https://gitter.im/decidim/decidim). We want you working on things you're excited about.
+
+#### **Did you find a bug?**
+
+* **Do not open up a GitHub issue if the bug is a security vulnerability in Decidim**, and instead send us an email to security [at] decidim.org.
```

Comment:
@andreslucena does the email address `security [at] decidim.org` exist?

Reply:
Yes, it works